content/TrueCommand/TCGettingStarted/Install/_index.md

```diff
@@ -178,36 +231,36 @@ The requirements to run TrueCommand in Docker Desktop for Windows are:
 * CPU support for VM Monitor Mode Extension (VT-c on Intel CPUs).
 * Hyper-V is enabled in Windows 10.
 * 4 GB memory at minimum.
-* [Docker Desktop](https://www.docker.com/products/docker-desktop) needs to be installed in Windows.
+* [Docker Desktop](https://www.docker.com/products/docker-desktop) must be installed in Windows.
 
 ### Enable Hyper-V
 
-To enable Hyper-V, click on Windows Start button and select or search for *Apps & Features*.
-Select *Programs and Features* under **Related settings** and *Turn Windows Features on or off* (Administrator action).
-Set the Hyper-V option and click *Ok*.
- You will need to restart your restart your computer for the change to take effect.
+Open the Start menu and search for Apps & Features.
+Select Programs and Features under Related settings and Turn Windows Features on or off (Administrator action).
+Set the Hyper-V option and click **Ok**.
+Restart your computer for the change to take effect.
 
 ![EnableHyperV](/images/TrueCommand/EnableHyperV.png "Enable HyperV")
 
-Alternatively, Hyper-V can be enabled with the **Powershell**.
-To do this, run Powershell as a Windows Administrator and enter `Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Hyper-V -All`.
-If an error is returned that says the command could not be found, verify that you are running PowerShell as the Administrator.
+Alternatively, you can enable Hyper-V with Powershell.
+Run Powershell as a Windows Administrator and enter `Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Hyper-V -All`.
+If Powershell returns an error saying it couldn't find the command, verify that you are running PowerShell as the Administrator.
 After the command successfully runs, reboot the computer.
 
 ![PowershellHyperVCLI](/images/TrueCommand/PowershellHyperVCLI.png "Powershell HyperV CLI")
 
 ### Install Docker Desktop
 
-Open Docker Hub and click *[Get Docker](https://hub.docker.com/editions/community/docker-ce-desktop-windows/)*.
+Open Docker Hub and click **[Get Docker](https://hub.docker.com/editions/community/docker-ce-desktop-windows/)**.
 
 ![DownloadDockerDesktop](/images/TrueCommand/DownloadDockerDesktop.png "Download Docker Desktop")
 
 Run the installer after the download completes.
-When the installer is finished, reboot the system.
+When the installer finishes, reboot the system.
 
 {{< expand "Different Admin accounts?" "v" >}}
-If the admin account is different from your Windows user account, the user must be added to the `docker-users` group.
-Run *Computer Management* as an administrator and go to **Local Users and Groups > Groups > docker-users** to add the user to the group.
+If the admin account is different from your Windows user account, you must add the user to the `docker-users` group.
+Run Computer Management as an administrator and go to **Local Users and Groups > Groups > docker-users** to add the user to the group.
```

Comment:
Why is Computer Management capitalized? it is not a proper name. Is this a Windows wizard? if it is and that is what it is named then use bold emphasis. If it is neither no capitalization required.